src/DataViews/ModulesDataViewTest.cpp

```diff
@@ -0,0 +1,297 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_format.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <filesystem>
+#include <string>
+
+#include "ClientData/ProcessData.h"
+#include "DataViews/DataView.h"
+#include "DataViews/ModulesDataView.h"
+#include "DisplayFormats/DisplayFormats.h"
+#include "MockAppInterface.h"
+#include "OrbitBase/File.h"
+#include "OrbitBase/ReadFileToString.h"
+#include "OrbitBase/TemporaryFile.h"
+#include "OrbitBase/TestUtils.h"
+#include "module.pb.h"
+
+using orbit_client_data::ModuleData;
+using orbit_client_data::ModuleInMemory;
+using orbit_grpc_protos::ModuleInfo;
+
+namespace {
+
+constexpr size_t kNumModules = 3;
+constexpr std::array<uint64_t, kNumModules> kStartAddresses{0x1000, 0x2000, 0x3000};
+constexpr std::array<uint64_t, kNumModules> kEndAddresses{0x1100, 0x2100, 0x3100};
+constexpr std::array<uint64_t, kNumModules> kSizes{100, 200, 300};
+const std::array<std::string, kNumModules> kNames{"module_abc", "module_abc", "module_xyz"};
+const std::array<std::string, kNumModules> kFilePaths{
+    "/usr/subpath/to/module_abc", "/local/subpath/to/module_abc", "/usr/subpath/to/module_xyz"};
+const std::array<std::string, kNumModules> kBuildIds{"build_id_0", "build_id_1", "build_id_2"};
+
+// ModulesDataView also has column index constants defined, but they are declared as private.
+constexpr int kColumnName = 0;
+constexpr int kColumnPath = 1;
+constexpr int kColumnAddressRange = 2;
+constexpr int kColumnFileSize = 3;
+constexpr int kColumnLoaded = 4;
+constexpr int kNumColumns = 5;
+
+std::string GetExpectedDisplayAddressRangeByIndex(size_t index) {
+  return absl::StrFormat("[%016" PRIx64 " - %016" PRIx64 "]", kStartAddresses[index],
```

Comment:
Nit: `PRIx64` is not required with abseil. Just `%016x` should be fine.

Reply:
Done! Thanks!

Reply:
Should I use `%016x` or `%016llx`? Seems that `PRIx64` is expanded as `llx`.

Reply:
`llx` is not needed because automatically detects the size of the type. `%016x` should be fine.

Reply:
Thanks for the explanation!